lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -2989,7 +2989,7 @@ def option_values_dispatch(o, str, words)
         option_values_target_addrs().each do |addr|
           res << addr
         end
-      when 'LHOST'
+      when 'LHOST', 'SRVHOST'
```

Comment:
```SRVHOST``` modules probably won't have an ```RHOST```, so the first case of the ```if``` doesn't really apply.